### PR TITLE
Jos/shepherd

### PR DESCRIPTION
**Pull request by JonathanS77**

### CODE REVIEW

File: adapted-articles/S/Shepherd.xml [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
+<p>The NT imagery comes from an OT and Palestinian background. In the Jewish economy, the shepherd who was taking care of a flock of sheep or goats held a responsible position. Great flocks had to be moved from place to place, and it was necessary to protect them from wild animals and robbers. Because of the fundamental role of shepherds in the ancient world, the word “shepherd” became a common term for a ruler. The kings of Assyria, Babylon, and Egypt were often referred to as shepherds who protected their people. This imagery formed the background for the OT, where the same usage is found. God is pictured as the shepherd of Israel, concerned for every aspect of his people’s welfare. Rulers and leaders of the people are often referred to as shepherds (<a href="?bref=Num.27.17">Nm 27:17</a>; <a href="?bref=1Kgs.22.17">1 Kgs 22:17</a>; <a href="?bref=Jer.10.21">Jer 10:21</a>; <a href="?bref=Jer.12.10">12:10</a>; <a href="?bref=Jer.22.22">22:22</a>; <a href="?bref=Jer.23.1-2">23:1-2</a>).</p>
```
Comment (joannapierquin) [inline adapted-articles/S/Shepherd.xml]: the shepherd who was taking care of a flock of sheep --> the shepherd who took care of a flock of sheep

**Review by joannapierquin — COMMENTED** [review]
LGTM

**Comment by JonathanS77** [comment]
@jgonz3 Hi! Here's a test we made on the Shepherd article. Please feedback.

**Comment by jgonz3** [comment]
> @jgonz3 Hi! Here's a test we made on the Shepherd article. Please feedback.

The adaptation looks good to me.

My only comment concerns the changelog:

The changes you made to the “Shepherd” article seem relevant and trivial to me, and could be documented as simply as saying “6 words/expressions have been changed for simplification sake.” (Counting can be approximate for larger articles, as the goal is to give an idea of the volume that has been reworked.)

I tend to only keep significant changes in the “Major Changes” section, such as a dropped paragraphs, or when ambiguous sentences had to be rephrased. That way linguists that will lean on our work can make informed choices.

**Comment by mvahowe** [comment]
Thanks @jgonz3! It's true that we don't need to document every change because github shows us what was changed anyway. I think we can go with these changelogs for now, and simplify going forward.

**Review by jgonz3 — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**